hunting-buddy.lic

```diff
@@ -78,13 +78,11 @@ class HuntingBuddy
     bput("stow my #{description}", 'You put')
   end
 
-  def check_prebuffs
-    return unless @settings@settings.pre_hunt_buff 
-    return unless !@settings.waggle_sets.nil?
+  def check_prehunt_buffs
```

Comment:
I like `prehunt_buffs` over `prebuffs`. What do you think about changing the waggle set name and room setting name to match?